pkg/process/events/listener_linux_test.go

```diff
@@ -0,0 +1,188 @@
+package events
+
+import (
+	"context"
+	"errors"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/process/events/model"
+	"github.com/DataDog/datadog-agent/pkg/security/api"
+	"github.com/DataDog/datadog-agent/pkg/security/api/mocks"
+)
+
+// assertProcessEvents compares two ProcessEvents. Two events can't be compared using directly assert.Equal
+// due to the embedded time fields
+func assertProcessEvents(t *testing.T, expected, actual *model.ProcessEvent) {
+	assert.Equal(t, expected.EventType, actual.EventType)
```

Comment:
you may want to add `t.Helper()` here so that stack shows the actual test spot, not the spot inside this helper.

Reply:
That's a really good suggestion! Added in https://github.com/DataDog/datadog-agent/pull/12099/commits/564beab9caa3c59be70329e66e06562877c63163.